csharp/ql/src/semmle/code/csharp/ir/implementation/raw/internal/TranslatedCall.qll

```diff
@@ -0,0 +1,333 @@
+import csharp
+private import semmle.code.csharp.ir.implementation.Opcode
+private import semmle.code.csharp.ir.implementation.internal.OperandTag
+private import InstructionTag
+private import TranslatedElement
+private import TranslatedExpr
+private import semmle.code.csharp.ir.Util
+
+/**
+ * The IR translation of a call to a function. The call may be from an actual
+ * call in the source code, or could be a call that is part of the translation
+ * of a higher-level constructor (e.g. the allocator call in a `NewExpr`).
+ */
+abstract class TranslatedCall extends TranslatedExpr {
+  override final TranslatedElement getChild(int id) {
```

Comment:
`id` -> `index` ?

Reply:
Should I refactor it in all the files?

Reply:
There is no requirement that the `id` range be contiguous. It often is, but for constructs where some children are optional, those children (and their `id`s) won't exist.